servant/src/Servant/Utils/Links.hs

```diff
@@ -74,7 +74,9 @@
 -- >>> safeLink api bad_link
 -- ...
 --     Could not deduce (Or
---                         (IsElem' (Delete '[JSON] ()) (Get '[JSON] Int))
+--                         (IsElem'
+--                            (Verb 'Network.HTTP.Types.Method.DELETE 200 '[JSON] ())
```

Comment:
I would re-export all the `StdMethod`s (`DELETE`, `GET`, etc.) to avoid the fully-qualified identifiers in type errors and thereby make them better readable.